macros/src/main/scala/com/gu/contentapi/circe/CirceScroogeMacros.scala

```diff
@@ -110,7 +113,6 @@ object CirceScroogeMacros {
     val tree = q"""{
       _root_.io.circe.Decoder.instance((cursor: _root_.io.circe.HCursor) => for (..${params.map(_._2)}) yield $apply(..${params.map(_._1)}))
     }"""
-    //println(showCode(tree))
```

Comment:
I am suggesting we keep the code commented, as this is useful for debugging. the hard part of the debug session is not to add the `println` but to remember where to put it and not to forget the `showCode ` instruction 😄